README.md

```diff
@@ -142,6 +143,25 @@ complete URL by setting `RUBY_BUILD_MIRROR_PACKAGE_URL`. It behaves the same as
 The default ruby-build download mirror is sponsored by
 [Basecamp](https://basecamp.com/).
 
+#### More flexible mirror URL
+
+For more flexible mirror URL, you can export a shell function to output the
+desired mirror URL, as shown below:
+
+```sh
+# There are two arguments:
+#   1st arg: the original URL without checksum
+#   2nd arg: the checksum
+mirror_cmd() {
+  echo "${1/cache.ruby-lang.org/mirror.example.com}"
+}
+
+export -f mirror_cmd
+export RUBY_BUILD_MIRROR_CMD=mirror_cmd
```

Comment:
This seems quite fragile and I guess only works if the original shell is Bash too.
Isn't RUBY_BUILD_MIRROR_URL enough?

Reply:
Yes, the example only works for bash. I'll update the example by using a more compatible way.